Doc/whatsnew/3.8.rst

```diff
@@ -394,6 +394,9 @@ Deprecated
 
   (Contributed by Serhiy Storchaka in :issue:`33710`.)
 
+* The :meth:`~isAlive()` method of :class:`threading.Thread` has been deprecated.
+
```

Comment:
Please remove this empty line, so both setences can be rendered on the same line.